### PR TITLE
Moving transport transactions page and adding it to menu

### PR DESCRIPTION
**Pull request by timbussmann**
Is the redirect configured correctly?

### CODE REVIEW

File: nservicebus/transports/transactions.md [hunk 1]
```diff
@@ -31,7 +33,7 @@ NServiceBus offers four levels of guarantees with regards to message processing.
...
 
-|  | [Transaction scope (Distributed transaction)](/nservicebus/messaging/transactions.md#transactions-transaction-scope-distributed-transaction) | [Transport transaction - Sends atomic with Receive](/nservicebus/messaging/transactions.md#transactions-transport-transaction-sends-atomic-with-receive)  | [Transport transaction - Receive Only](/nservicebus/messaging/transactions.md#transactions-transport-transaction-receive-only) | [Unreliable (Transactions Disabled)](/nservicebus/messaging/transactions.md#transactions-unreliable-transactions-disabled) |
+|  | [Transaction scope (Distributed transaction)](/nservicebus/transports/transactions.md#transactions-transaction-scope-distributed-transaction) | [Transport transaction - Sends atomic with Receive](/nservicebus/transports/transactions.md#transactions-transport-transaction-sends-atomic-with-receive)  | [Transport transaction - Receive Only](/nservicebus/transports/transactions.md#transactions-transport-transaction-receive-only) | [Unreliable (Transactions Disabled)](/nservicebus/transports/transactions.md#transactions-unreliable-transactions-disabled) |
```
Comment (timbussmann) [inline nservicebus/transports/transactions.md]: wouldn't `[Transaction scope (Distributed transaction)](#transactions-transaction-scope-distributed-transaction)` work too?

**Comment by timbussmann** [comment]
ping @SimonCropp

File: menu/menu.yaml [hunk 2]
```diff
@@ -182,6 +182,7 @@
           Url: "nservicebus/transports"
           Articles:
             - "nservicebus/transports/queuecreation"
+        - "nservicebus/transports/transactions"
```
Comment (weralabaj) [inline menu/menu.yaml]: seems like sth is wrong with tabs or spaces here

File: nservicebus/transports/transactions.md (hunk 1 above)
Comment (weralabaj) [inline nservicebus/transports/transactions.md]: it should

**Comment by weralabaj** [comment]
why do we move this page?

File: menu/menu.yaml (hunk 2 above)
Comment (timbussmann) [inline menu/menu.yaml]: no, I intended this page to be on the "root" of the transports menu, not inside "Extending"

**Comment by timbussmann** [comment]
> why do we move this page?

since it's only shown on the transports menu, I think it makes sense to move it there?

**Pull request merged**